SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialAuthorityTest/SpatialAuthorityTest.cpp

```diff
@@ -468,6 +499,117 @@ void ASpatialAuthorityTest::PrepareTest()
 	}
 }
 
+void ASpatialAuthorityTest::CheckDoesNotMigrate(ASpatialAuthorityTestActor* Actor, int ServerId)
```

Comment:
Perhaps ServerId should be ExpectedServerId ?

Reply:
Done